mapview/view.py

```diff
@@ -512,6 +514,9 @@ def sync_to(self, other):
     def __init__(self, **kwargs):
         from kivy.base import EventLoop
         EventLoop.ensure_window()
+        # CACHE_DIR must be in a place where the user has read/write permissions
+        self.cache_dir = self.cache_dir if platform != 'ios' else join(
```

Comment:
this works, but i would rather just have
```python
if platform == 'ios':
    self.cache_dir = …
```

Reply:
Sure thing. How about 
```
        if platform == 'ios':
            app = App.get_running_app()
            self.cache_dir = join(app.user_data_dir, self.cache_dir)
```

Reply:
Yes, that yould be ideal imho.

Reply:
Yes, that yould be ideal imho.

Reply:
Done! Thanks for the input.